_components/code-component.md

```diff
@@ -0,0 +1,221 @@
+---
+title: Code component
+layout: article
+section: Utility Components
+---
+
+## Description
+
+{{site.data.tenant.name}} Code component executes a piece of code. It is available
```

Comment:
Have you seen some update work done by @zubairov https://github.com/elasticio/code-component/pull/9 ?